src/Application/Itmo.Dev.Asap.Github.Application.Contracts/Submissions/ErrorMessages/ActivateErrorMessage.cs

```diff
@@ -1,25 +1,31 @@
-﻿namespace Itmo.Dev.Asap.Github.Application.Contracts.Submissions.ErrorMessages;
+﻿using Itmo.Dev.Asap.Github.Application.Abstractions.Octokit.Notifications;
 
-public class ActivateErrorMessage : ErrorMessage
+namespace Itmo.Dev.Asap.Github.Application.Contracts.Submissions.ErrorMessages;
+
+public class ActivateErrorMessage : IErrorMessage
 {
     private const string Title = "Error occured while processing Activate command";
+    private const string IssuerNotFoundMessage = $"{Title} \n Issuer was not found";
+    private const string SubmissionNotFoundMessage = $"{Title} \n Submission was not found";
+
+    private readonly string _message;
 
     private ActivateErrorMessage(string message)
     {
-        Message = message;
+        _message = message;
     }
 
-    public static ActivateErrorMessage IssuerNotFound()
+    public static ActivateErrorMessage IssuerNotFound => new(IssuerNotFoundMessage);
+
+    public static ActivateErrorMessage SubmissionNotFound => new(SubmissionNotFoundMessage);
```

Comment:
так ты энивей каждый раз новые объекты создаёшь 

```suggestion
    public static readonly ActivateErrorMessage IssuerNotFound = new(IssuerNotFoundMessage);

    public static readonly ActivateErrorMessage SubmissionNotFound = new(SubmissionNotFoundMessage);
```